lib/info.js

```diff
@@ -426,6 +426,21 @@ const getM3U8 = async(url, options) => {
   return formats;
 };
 
+// Check for updates.
+const { version } = require('../package.json');
```

Comment:
This breaks when bundling with webpack, not sure why. It imports the package.json from my project root, instead of ytdl-core's package.json. Workaround for me was to add a "version" field to my package.json, so it doesn't error out with 

```
TypeError: pkg.version is undefined
```

Might very well be me configuring webpack incorrectly, but thought I'd mention it here.

Repo to reproduce the issue: https://github.com/stoically/humanetube/

Reply:
weird. i'd ask webpack about it. otherwise, not sure what's causing webpack to import the wrong `package.json`

node's `require` works a bit different for relative paths, it looks for files relatively to the file it is required from.

Reply:
I bundled it with Browserify and it worked fine; I don't know enough about Webpack to help you further.